src/components/sdk/DeviceSelection/CameraSelection/index.tsx

```diff
@@ -5,21 +5,28 @@ import React from 'react';
 
 import { useVideoInputs } from '../../../../providers/DevicesProvider';
 import { useMeetingManager } from '../../../../providers/MeetingProvider';
+import { DeviceConfig } from '../../../../types';
 import DeviceInput from '../DeviceInput';
 
 interface Props {
   /** The message that will be shown when no camera devices are found. */
   notFoundMsg?: string;
   /** The label that will be shown for camera selection, it defaults to "Camera source". */
   label?: string;
+  /** A boolean that determines whether or not to include additional sample video devices, such as "None", "Blue", "SMTP Color Bars". This will be deprecated in the next major version. */
```

Comment:
Could you please mention that the default is true? Also, request to update the documentation here and elsewhere that with prop there will be no option to add any sample devices in next major release.

Reply:
Done. dont the comments from these props go directly into storybook documentation? Is that good enough?

Reply:
Yeah, I meant expanding the comment here itself no other documentation.